common/src/test/scala/io/phdata/pulse/common/SolrServiceTest.scala

```diff
@@ -132,25 +131,25 @@ class SolrServiceTest extends FunSuite with BaseSolrCloudTest {
     val collectionName = TestUtil.randomIdentifier()
     solrService.createCollection(collectionName, 1, 1, "testconf", null)
 
-    val document1 = DocumentConversion.toSolrDocument(
-      new LogEvent(None,
-                   "ERROR",
-                   "1970-01-01T00:00:00Z",
-                   "ERROR",
-                   "message",
-                   "thread oxb",
-                   Some("Exception in thread main"),
-                   None))
-
-    val document2 = DocumentConversion.toSolrDocument(
-      new LogEvent(None,
-                   "ERROR",
-                   "1970-01-01T00:00:00Z",
-                   "ERROR",
-                   "message",
-                   "thread oxb",
-                   Some("Exception in thread main"),
-                   None))
+    val document1 = DocumentConversion.mapToSolrDocument(
+      Map(
+        "category"   -> "ERROR",
+        "timestamp"  -> "1970-01-01T00:00:00Z",
+        "level"      -> "ERROR",
+        "message"    -> "message",
+        "threadName" -> "thread oxb",
+        "throwable"  -> "Exception in thread main"
+      ).asInstanceOf[Map[String, String]])
+
+    val document2 = DocumentConversion.mapToSolrDocument(
+      Map(
+        "category"   -> "ERROR",
+        "timestamp"  -> "1970-01-01T00:00:00Z",
+        "level"      -> "ERROR",
+        "message"    -> "message",
+        "threadName" -> "thread oxb",
+        "throwable"  -> "Exception in thread main"
+      ).asInstanceOf[Map[String, String]])
```

Comment:
Why the cast? I don't think it's necessary

Reply:
Without casting `mapToSolrDocument` complains about the `None`

Reply:
Can you share that error? Solr should be creating its own ids, and I believe that endpoint takes a `SolrDocument` type, so this is odd.